packages/webapp/src/components/Navigation/SideMenu/index.jsx

```diff
@@ -233,7 +233,7 @@ const PureSideMenu = ({
         isOpen={isDrawerOpen}
         onClose={onDrawerClose}
         fullHeight
-        responsiveModal={false}
+        desktopVariant={DesktopDrawerVariants.DRAWER}
```

Comment:
@antsgar I don't know if you remember back well enough to when the `responsiveModal` property was defined, but it doesn't seem necessary in this context because the Drawer is never a modal in mobile view anyway! Maybe Drawer or SideMenu also got a refactor since originally written?

Reply:
Yeah since this only renders in mobile view I don't think the prop makes sense, I'm not sure why I passed it along initially or if something got refactored along the way